ui/panels/emailer.php

```diff
@@ -195,20 +205,90 @@
 	</div>
 </div>
 
+<?php //Set Different From email and Reply-to text depending on Pro delivery status of the form
+    $enhanced_delivery = \calderawp\calderaforms\pro\container::get_instance()->get_settings()->get_enhanced_delivery();
```

Comment:
This has to go after conditional, `\calderawp\calderaforms\pro\container::get_instance()` is not defiend if that fails